pr-data.csv

```diff
@@ -1736,15 +1736,23 @@ https://github.com/apache/flink,e246a05cd898e16758a52335250571e165a5148a,flink-t
+https://github.com/apache/fory,9295e58ae706a3734f6fa80704c41502ba73131e,fory-core,org.apache.fory.collection.LazyMapTest.testMap,ID,,,
+https://github.com/apache/fory,9295e58ae706a3734f6fa80704c41502ba73131e,fory-core,org.apache.fory.ForyCopyTest.mutableObjectCopyTest,ID,,,
+https://github.com/apache/fory,9295e58ae706a3734f6fa80704c41502ba73131e,fory-core,org.apache.fory.resolver.MetaContextTest.testShareClassDefCompatible,ID,,,
+https://github.com/apache/fory,9295e58ae706a3734f6fa80704c41502ba73131e,fory-core,org.apache.fory.resolver.MetaContextTest.testShareClassName,ID,,,
+https://github.com/apache/fory,9295e58ae706a3734f6fa80704c41502ba73131e,fory-core,org.apache.fory.serializer.collection.CollectionSerializersTest.tesPriorityQueueSerializer,ID,,,
+https://github.com/apache/fory,9295e58ae706a3734f6fa80704c41502ba73131e,fory-core,org.apache.fory.serializer.collection.CollectionSerializersTest.testCollectionFieldSerializers,ID,,,
+https://github.com/apache/fory,9295e58ae706a3734f6fa80704c41502ba73131e,fory-core,org.apache.fory.serializer.collection.CollectionSerializersTest.testCollectionFieldSerializersCopy,ID,,,
+https://github.com/apache/fory,9295e58ae706a3734f6fa80704c41502ba73131e,fory-core,org.apache.fory.serializer.ProtocolInteroperabilityTest.testComplexCollection,ID,,,
 https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.codegen.ExpressionVisitorTest.testTraverseExpression,ID,Accepted,https://github.com/apache/fury/pull/1968,
-https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.collection.LazyMapTest.testMap,ID,,,
-https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.FuryCopyTest.mutableObjectCopyTest,ID,,,
-https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.resolver.MetaContextTest.testShareClassDefCompatible,ID,,,
-https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.resolver.MetaContextTest.testShareClassName,ID,,,
-https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.serializer.collection.CollectionSerializersTest.tesPriorityQueueSerializer,ID,,,
-https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.serializer.collection.CollectionSerializersTest.testCollectionFieldSerializers,ID,,,
-https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.serializer.collection.CollectionSerializersTest.testCollectionFieldSerializersCopy,ID,,,
-https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.serializer.ProtocolInteroperabilityTest.testComplexCollection,ID,,,
+https://github.com/apache/fury,9295e58ae706a3734f6fa80704c41502ba73131e,fury-core,org.apache.fury.collection.LazyMapTest.testMap,ID,MovedOrRenamed,,
```

Comment:
If you put `MovedOrRenamed`, it wants a URL to the commit that moved or renamed the test. In this case, since the repo name change, can you try to put just the note `Repo was renamed from apache/fury to apache/fory`; you need not find the date when it happened (as it may not be recorded in Git anyhow).